common/common.go

```diff
@@ -464,8 +464,8 @@ func BlockIngestor(c *BlockCache, rep int) {
 			c.Sync()
 			Log.Info("Waiting for zcashd height to reach Sapling activation height ",
 				"(", c.GetFirstHeight(), ")...")
-			Time.Sleep(20 * time.Second)
-			return
```

Comment:
The actual bug is that here we're exiting the block ingestor thread, which is supposed to run forever. Nothing ever starts it up again.